.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for MetaMorpheus
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the feature you'd like**
+What would you like to see done in MetaMorpheus?
```

Comment:
Alternate text: "We love to create new features for our users. Please send us your great ideas and explain why they would be useful. Let us know if youhave data (real or synthetic) that we could use to test if our solution is achieving the desired result."

Reply:
added it!